### ISSUE TITLE
NVidia Shield : No DTS or DD passthrough when using USB Sound Stick.

### ISSUE THREAD

**Issue by VimesUK** [koying/SPMC#162]
As noted over in the SPMC thrtead within the Kodi forums I haver a Nvidia Shield.
Connected to that is a Turtle Beach Audio Advantage Micro II USB sound stick with a SPDIF Optical out.
This connects to my Logitech Z5500 speakers optically and provides working DTS and DD pass through when using Kodi 16.1 RC2.
No issues at all are noted.

When using the latest SPMC build it does not matter whether I select PCM Android, RAW Passthrough or IEC Passthrough if I try and use it when playing a DD or DTS file I then get a black screen with the video not playing until I exit the film and back to the SPMC UI. If I turn off passthrough with SPMC then I get the films to play fine but, obviously, with no DTS or DD.

Going back to Kodi 16.1 RC 2 (worked with 16 also) and it works fine with perfect passthrough.

The sound stick also worked great with a Windows based HTPC and a Pi 3 using Open Elec etc.

**Comment by koying** [comment]
Please try http://download.semperpax.com/spmc/android-arm/SPMC-20160330-59124a3-spmc-jarvis-armeabi-v7a.apk to confirm the issue is solved

**Comment by VimesUK** [comment]
The test build of SPMC 16.2-0 worked fine with IEC sound and Passthrough enabled. 
I purposely set the volume low on the Shield before I started SPMC with the new build. When selecting a film it raised the volume to full on the Shield and the sound was perfect.
The film was stopped and the volume on the Shield was returned to where I had set it previously.

The above repeats as required as pass through audio as expected.  

Many thanks for a speedy build and working solution
